src/AssemblySharedInfoGenerator/AssemblyInfoGenerator.csproj

```diff
@@ -63,8 +63,11 @@
   </PropertyGroup>
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
        Other similar extension points exist, see Microsoft.Common.targets.
+  -->
   <Target Name="BeforeBuild">
+    <Exec Command="$(SolutionDir)restorepackages.bat" IgnoreExitCode="true" />
```

Comment:
IgnoreExitCode should be false, so that when the batch file fails to execute the build is broken.